types/jspdf/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for jsPDF v1.1.135
+// Type definitions for jsPDF v1.1.136
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).